run.py

```diff
@@ -83,9 +84,15 @@ def train_and_test(epoch):
     train(epoch)
     test()
 
+    dummy_input = torch.randn(1, 1, 32, 32, requires_grad=True)
+    torch.onnx.export(net, dummy_input, "lenet.onnx")
+
+    onnx_model = onnx.load("lenet.onnx")
+    onnx.checker.check_model(onnx_model)
+
 
 def main():
-    for e in range(1, 16):
+    for e in range(1, 3):
```

Comment:
Does it cross 99% right after 3 epochs? If not do you mind keeping this the same.

Reply:
No, it doesn't I reduced the epochs for testing. I'm going to revert that.

Reply:
Now after letting it train for 16 epochs it reaches an accuracy of: 0.990800. With the added residual connection and the updated packages in `requirements.txt`